examples/README.md

```diff
@@ -54,3 +54,48 @@ aws s3api create-bucket --bucket <bucket-name> --region <region>
   <ecs-task-family> \
   <ecs-service-name>
 ```  
+
+## FireLens on ECS Fargate
+
+For running on Fargate, s3 file option is not supported for Fluent-bit config. Hence, we need to build the fluent-bit custom config image and then use its reference in our Firelens container definition.
+
+For building the custom fluent-bit image, clone the [amazon-ecs-firelens-examples](https://github.com/aws-samples/amazon-ecs-firelens-examples) and modify the contents of [extra.conf](https://github.com/aws-samples/amazon-ecs-firelens-examples/blob/mainline/examples/fluent-bit/config-file-type-file/extra.conf) in the amazon-ecs-firelens-examples repository. Post this run the following commands to build the custom fluent-bit image:-
```

Comment:
we can't pull the image directly from repo as mentioned here https://github.com/awslabs/aws-embedded-metrics-java/blob/master/examples/ecs-firelens/container-definitions.template.json#L21? manually clone is cumbersome. ?

Reply:
In both cases we are pulling the image directly from the "aws-for-fluent-bit" public repository.
The s3 option works directly by referencing it in the container definition. For AWS Fargate, s3 option is not supported so we have to go with the file type option.
If the file config file type is used, the config file value is the full path of the configuration file that exists either in the container image or on a volume that's mounted in the container. Source - https://docs.aws.amazon.com/AmazonECS/latest/developerguide/firelens-taskdef.html
Since the public container image does not have the extra.conf file, thus we need to manually push it to a manually created image on our own AWS account and then refer it in the container definition.